GRIDTHEME.md

```diff
@@ -0,0 +1,102 @@
+GridView Theme
```

Comment:
Not today but in a near future there is a possibility that this will change, I still need to think more about it.